internal/handlers/common/error.go

```diff
@@ -119,3 +119,25 @@ func ProtocolError(err error) (*Error, bool) {
 var (
```

Comment:
Interface checking should be at the very end of the file

Reply:
Fixed